spec/spec_helper.rb

```diff
@@ -50,11 +52,11 @@ def cork.string
 # running a PR on TravisCI
 def testing_env
```

Comment:
Is all of this needed since we don't use Travis CI?

Reply:
Removed the `travis` part but we need the configurations so that Danger can use it as an environment to run specs.